### ISSUE TITLE
check_uptime (new plugin)

### ISSUE THREAD

**Issue by wopfel** [monitoring-plugins/monitoring-plugins#1534]
I've created **check_uptime**, a Perl script showing and alarming about the uptime. Are you interested in a pull request?

```
# /usr/local/bin/check_uptime -w 10 -c 11      
CRITICAL: uptime is 5005860 seconds. |uptime=5005860;10;11;0
# /usr/local/bin/check_uptime -w 10 -c 11 -f
CRITICAL: uptime is 5005862 seconds. Running for 8 weeks, 1 day, 22 hours, 31 minutes and 2 seconds. |uptime=5005862;10;11;0
# /usr/local/bin/check_uptime -w 10 -c 11 -s
CRITICAL: uptime is 5005864 seconds. Running since 2018-03-30 14:58:53. |uptime=5005864;10;11;0
# /usr/local/bin/check_uptime -w 10 -c 11 -s -f
CRITICAL: uptime is 5005871 seconds. Running for 8 weeks, 1 day, 22 hours, 31 minutes and 11 seconds. Running since 2018-03-30 14:58:53. |uptime=5005871;10;11;0
```

Kind regards,
Bernd

**Comment by sni** [comment]
why not, please create a pull request and then we can have a look. We usually don't add any more new plugins, but i'd have a look at this since it is generic enough, of general use and hopefully quite simple.

**Comment by wopfel** [comment]
Please have a look at pull request #1538.

**Comment by wopfel** [comment]
Merged in 22f13aa. Thanks :-)
